Packs/Netmiko/Integrations/Netmiko/README.md

```diff
@@ -0,0 +1,119 @@
+
+
+## Netmiko SSH module integration
+
+  This integration provides ssh-based access to network devices, servers, and other appliances that support this method of configuration. For a complete list of supported platforms, please visit the below URL:
+
+  [Netmiko Platforms.md on Github](https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md)
+
+## Configuration Parameters
```

Comment:
Is this section necessary since you have these parameters listed in the following section.